desktop/apps/categories3/components/GeneFamilyNav.js

```diff
@@ -41,20 +48,53 @@ const GeneFamilyLink = styled.a`
   }
 `
 
-const GeneFamilyNav = ({ geneFamilies }) => {
-  return (
-    <ResponsiveSidebar>
-      <GeneFamilyList>
-        {geneFamilies.map(geneFamily =>
-          <GeneFamilyItem key={geneFamily.id}>
-            <GeneFamilyLink href={`#${geneFamily.id}`}>
-              {geneFamily.name}
-            </GeneFamilyLink>
-          </GeneFamilyItem>
-        )}
-      </GeneFamilyList>
-    </ResponsiveSidebar>
-  )
+class GeneFamilyNav extends React.Component {
+  constructor (props) {
+    super(props)
+    this.handleClick = this.handleClick.bind(this)
```

Comment:
You can take advantage of [classProperties](https://github.com/artsy/force/blob/master/.babelrc#L14) and omit the constructor completely. 

```
handleClick = (event) => {
  // do stuff -- fat arrow binds to the outer scope 
}

Reply:
Thanks for the reminder that we are using property initializers in Force — so long, `this… = this…bind(this)` :champagne: